src/lib/controllers/configuration-manager/generators/generate-private-key.ts

```diff
@@ -0,0 +1,11 @@
+import { pki } from 'node-forge';
+
+/**
+ * GENERATE_PRIVATE_KEY
+ * @param {number} bits: Defaults to 4096. RSA bits for generated key.
+ * @returns {Promise<string>} PrivateKeyPEM
+ */
+export function generatePrivateKey(bits = 4096) {
+	const keypair = pki.rsa.generateKeyPair({ bits, workers: 1 });
```

Comment:
Could just include the `pki.rsa.generateKeyPair()` straight into the `Buffer.from()`.

Reply:
done